fog/overseer/server/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+#![feature(proc_macro_hygiene, decl_macro)]
```

Comment:
do we really need nightly features for this?

can we avoid using unstable features if they aren't needed?

Reply:
Yes, we need these to enable [rocket_codegen](https://docs.rs/rocket_codegen/latest/rocket_codegen/), which provides the macros for `#[post(...)] `and `#[get(...)]`. [Full Service](https://github.com/mobilecoinofficial/full-service/blob/main/full-service/src/lib.rs#L5) uses these declarations as well.

Reply:
ok